test/fixtures/experimental/class-private-properties/asi-failure-computed/actual.js

```diff
@@ -1,4 +1,4 @@
 class Foo {
   #p = x
-  [#m] () {}
```

Comment:
`#4` should be a syntax error, as `#` needs to be followed by an IdentifierName.\

`[this.#x]` should be a ReferenceError when the class declaration is executed, as `#x` is "in scope" as a name during the evaluation of computed property names, but then `#x` cannot possibly be present on `this`.